### DISCUSSION TITLE
New feature idea: Run Macros from Best Practice Analyzer view

### DISCUSSION THREAD

**Discussion by l33smyth** [TabularEditor/TabularEditor3#1071] [Ideas]
Hi
I have a number of macros and a number of custom rules in the Best Practice Analyzer now that speed up my work massively (thank you!!)... It'd be great to be able to invoke them from a right-click menu on the BPA view, rather than having to "Go to Object" and doing so from the TOM Explorer. 

The gold-plated version of this would be being able to associate certain macros with a particular rules.

**Comment by otykier** [comment]
Great idea, thanks for suggesting!
